### PR TITLE
Fix user following

### LINKED ISSUE #6200: Couldn't follow 'User'! ~user is inaccessible
## Expected Behavior
No Message, Following detected
## Current Behavior
Following still working, but it will not detected

## InstaPy configuration

session.set_user_interact(amount=1, randomize=True, percentage=100, media='Photo')

        session.set_do_follow(enabled=True, percentage=100)

### LINKED ISSUE #6205: session.follow_by_tags ~user is inaccessible
## Expected Behavior
No error message, follow by tag detected
## Current Behavior

WARNING [2021-06-01 16:39:07] [XXXXX]  --> Couldn't follow 'XXX'!	~user is inaccessible

Same problem as before with "like by tags" (which has already been fixed)

user followed anyway, but no ID Tracking.

### PR DESCRIPTION
**Pull request by schealex**
## Description

Sometimes the following of users did not work and in addition if the follow was successful it could not be verified because of the follow button in the "promoted" users div.

Fixes #6200
Fixes #6205

## How Has This Been Tested?

Tested and working fine for me now

- [x] Test

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project, `black -t py34`
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by filiperivelli** [comment]
In my test, it was possible to follow a user and this follow was confirmed but just when I use `interact=False`.
I guess there is a problem in the like function either but I guess you will not fix this on this PR, am I correct?

**Comment by schealex** [comment]
> In my test, it was possible to follow a user and this follow was confirmed but just when I use `interact=False`.
> I guess there is a problem in the like function either but I guess you will not fix this on this PR, am I correct?

feel free to open an issue about this and if you give me a little more context and a sample configuration i can try to fix that in another seperate PR :)

File: instapy/browser.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def set_selenium_local_session(
 
 
 def proxy_authentication(browser, logger, proxy_username, proxy_password):
-    """ Authenticate proxy using popup alert window """
```
Comment (elulcao) [inline instapy/browser.py:161]: Hello 
Why this change from """[space] to """[a-zA-Z]?
is this just for pretty look?

File: instapy/unfollow_util.py [hunk 2]
```diff
@@ -1551,6 +1553,17 @@ def verify_action(
 
                     sleep(4)
                 elif retry_count == 3:
+                    user_link = "https://www.instagram.com/{}/".format(person)
+                    web_address_navigator(browser, user_link)
+                    follow_button_xp = read_xpath(
+                        "get_following_status", "follow_button_XP"
+                    )
+                    button = browser.find_element(By.XPATH, follow_button_xp)
```
Comment (elulcao) [inline instapy/unfollow_util.py]: Why not using `browser.find_element_by_xpath(follow_button_xp)`?
Did you miss the try/catch for the button.click()?

**Comment by elulcao** [comment]
CHANGELOG information is missing,
Could you change the PR to merge the change to dev branch,

**Comment by elulcao** [comment]
CHANGELOG information is missing,
Could you change the PR to merge the change to dev branch,

**Comment by schealex** [comment]
@elulcao changed code based on your review but here again why would you want to merge this to the dev branch? oO

**Review by elulcao — APPROVED** [review]

**Comment by adrielkirch** [comment]
I've downloaded your source code  to fix the 'inaccessible user' and it dind't work yet,  '~user is inaccessible'
What can I do to fix that ??
